src/lib/shared/utils.ts

```diff
@@ -70,9 +70,9 @@ export function base64StringToTransaction<T extends Transaction>(transactionByte
  * @param Query
  * @returns a base64 encoded string
  */
-export function queryToBase64String<T, Q extends Query<T>>(query: Q): string {
-  const queryBytes = query.toBytes()
-  return Buffer.from(queryBytes).toString('base64')
+export function queryToBase64String<T, Q extends Query<T> = Query<T>>(query: Q): string {
+  const queryBytes = query.toBytes();
+  return Buffer.from(queryBytes).toString('base64');
```

Comment:
@BeeJeeNinja please elaborate on these changes

Reply:
well, I guess we don't really need this